media/video/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Upload module"""
+import logging
+from bson import ObjectId
+from flask import request
+import json
+from media import get_collection, blueprint
+from flask import Blueprint
+
+bp = Blueprint('project', __name__)
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, ObjectId):
+            return str(o)
+        return json.JSONEncoder.default(self, o)
+
+
+@bp.route('/project', methods=['POST'])
+def create_video_editor():
+    if request.method == 'POST':
+        files = request.files.to_dict(flat=False)['media']
+        user_agent = request.headers.environ['HTTP_USER_AGENT']
+
+        return create_video(files, user_agent)
+
+
+@bp.route('/project/<path:video_id>', methods=['GET', 'PUT', 'DELETE'])
+def process_video_editor(video_id):
+    """Keep previous url for backward compatibility"""
+    if request.method == 'GET':
+        return get_video(video_id)
+    if request.method == 'PUT':
+        return update_video(video_id, request.form)
+    if request.method == 'DELETE':
+        return delete_video(video_id)
+
+
+def delete_video(video_id):
+    video = get_collection('video')
+    video.remove({'_id': ObjectId(video_id)})
+    return 'delete successfully'
+
+
+def update_video(video_id, updates):
+    video = get_collection('video')
+    updates = {"$set": {"chunkSize": 0}}
+    update = video.update_one({'_id': ObjectId(video_id)}, updates)
+    return 'update successfully'
+
+
+def create_video(files, agent):
+    """Save video to storage and create records to databases"""
+    video = get_collection('video')
+    docs = []
+    for file in files:
+        doc = {
+            'metadata': None,
+            'client_info': agent,
+            'version': 1,
+            'processing': False,
+            "parent": None,
+            'thumbnails': {}
+        }
+        docs.append(doc)
+    video.insert_many(docs)
+    return JSONEncoder().encode(docs)
```

Comment:
Not for this PR, but for next PRs we should return correct headers with mimetype and status.
`Response(JSONEncoder().encode(docs), status=200, mimetype='application/json')`

Reply:
you can use `from flask import jsonify` and then `return jsonify(docs)` that should also handle mimetype

Reply:
hi, @petrjasek 
I see the jsonify can not convert Objectid, or I did something wrong.
So i use json_util to convert it, is it right?